rocAL/rocAL/source/meta_data/meta_node_crop_mirror_normalize.cpp

```diff
@@ -45,8 +45,8 @@ void CropMirrorNormalizeMetaNode::update_parameters(pMetaDataBatch input_meta_da
     _dstImgHeight = _meta_crop_param->croph_arr;
     _x1 = _meta_crop_param->x1_arr;
     _y1 = _meta_crop_param->y1_arr;
-    _src_width = _node->get_src_width();
-    _src_height = _node->get_src_height();
+    // _src_width = _node->get_src_width(); // TODO - Commenting for now to be replaced with ROI
+    // _src_height = _node->get_src_height();
```

Comment:
Please check if this changed is intended

Reply:
Yes, will be fixed in future PR